packages/k2-alpine/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { forwardRef, PropsWithChildren, useMemo } from 'react'
+import {
+  Platform,
+  StyleProp,
+  StyleSheet,
+  TouchableOpacity,
+  ViewStyle
+} from 'react-native'
+import { useDripsyTheme as useTheme } from 'dripsy'
+import { Text, View } from '../Primitives'
+import { Icons } from '../../theme/tokens/Icons'
+import {
+  colors,
```

Comment:
is it possible to not use colors directly?

Reply:
good point. fixed.